corehq/apps/sms/migrations/0024_add_is_two_way.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from corehq.sql_db.operations import HqRunPython
+from django.db import migrations, models
+import datetime
+
+
+def set_pending_verification_numbers(apps, schema_editor):
+    PhoneNumber = apps.get_model('sms', 'PhoneNumber')
+    db_alias = schema_editor.connection.alias
+    PhoneNumber.objects.using(db_alias).filter(verified=False).update(is_two_way=False, pending_verification=True)
```

Comment:
Should `verified=True` imply `is_two_way=True` and should it be added to this migration?